test/atm/global-workflow/jjob_post.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -x
+bindir=$1
+srcdir=$2
+
+# prepare stuff from previous cycle
+
+# run jjob
+export HOMEgfs=$srcdir/../../ # TODO: HOMEgfs had to be hard-coded in config
+echo $HOMEgfs
+
+export EXPDIR=$bindir/test/atm/global-workflow/testrun/experiments/gdas_test
+export PDY=20210323
+export cyc=18
+export CDATE=${PDY}${cyc}
+export ROTDIR=$bindir/test/atm/global-workflow/testrun/ROTDIRS
+export CDUMP=gdas
+export DATAROOT=$bindir/test/atm/global-workflow/testrun/RUNDIR
+export COMIN_GES=${bindir}/test/atm/bkg
+export pid=${pid:-$$}
+export jobid=$pid
+export COMROOT=$DATAROOT
+
+# prepare background from previous cycle
```

Comment:
I suspect some of the comments below can be removed?

Reply:
Thanks.  Extraneous comments removed at [837012b](https://github.com/NOAA-EMC/GDASApp/pull/244/commits/837012b34ad2990e80f640e079985ef2e18492ca)